helmfile/charts/networkpolicy-service/templates/thanos/allow-thanos-buketweb.yaml

```diff
@@ -0,0 +1,38 @@
+{{ if .Values.thanos.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: thanos-bucketweb
+  namespace: thanos
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: bucketweb
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: monitoring
```

Comment:
Does prometheus talk to bucket web? Is it some metrics or?

Reply:
Unsure, i have just split the np for monitoring and thanos. So i have not really looked into them.

Reply:
This is for retrieving thanos bucket metrics, such as `thanos_bucket_blocks_*`